app/helpers/api/ordapi/getOrder.js

```diff
@@ -0,0 +1,40 @@
+import { getData } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { orderApiUrl } from '../../../config';
+
+const getOrderEndpoint = orderId => `${orderApiUrl}/api/v1/orders/${orderId}`;
+
+const createServiceRecipientsDict = (serviceRecipients = []) => {
+  const reducer = (dict, serviceRecipient) => (
+    {
+      ...dict,
+      [serviceRecipient.odsCode]: serviceRecipient,
+    });
+
+  return serviceRecipients.reduce(reducer, {});
+};
+
+const isOneOff = orderItem => orderItem.catalogueItemType === 'AssociatedService'
```

Comment:
Probably don't need to but might want to consider ignoring the case of the comparisons here.